app/assets/stylesheets/stores.css.scss

```diff
@@ -2,6 +2,16 @@
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
+.create_store-wrapper h1{
+  font-size: 3em;
+  color: #E67E22;
+}
+.create_store-wrapper label{
```

Comment:
Selector `create_store-wrapper` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space